### ISSUE TITLE
cpp, Simplified Chinese garbled

### ISSUE THREAD

**Issue by fairycn** [EpicGamesExt/raddebugger#230]
cpp, Simplified Chinese garbled
As shown in the following figure when there is Simplified Chinese in the code, the relevant characters cannot be displayed normally
![image](https://github.com/EpicGamesExt/raddebugger/assets/74580422/d2d20e46-ccd4-4da8-8960-9d545f446b47)

**Comment by skejeton** [comment]
Looks like the font doesn't support Chinese glyphs and the font renderer doesn't fallback. Does it work if you change to a font with Chinese glyphs in it?

**Comment by fairycn** [comment]
Thank you, to pass this method to solve the font display problem, but found that the code file in the first line more than an unknown character (not caused by the font file, you can refer to the previous picture, not set the body of the code file at the beginning of the contents of the content compared to the source code more than some of the content.)
![image](https://github.com/EpicGamesExt/raddebugger/assets/74580422/7950144b-3996-43f6-905a-0f4a5df2b390)

**Comment by fairycn** [comment]
Yes, it means what you understand it to mean (the code file is generated by the vs 2022 default settings)

**Comment by dbechrd** [comment]
Feature request: It would be nice if there were some way to set the font via the UI, and preferably choose from the standard list of installed fonts on Windows. Having to find a config file and copy the .ttf there manually was not easy to discover.
